lib/client/api.go

```diff
@@ -1047,9 +1047,25 @@ func (tc *TeleportClient) ConnectToProxy() (*ProxyClient, error) {
 	return proxyClient, nil
 }
 
-// Logout locates a certificate stored for a given proxy and deletes it
+// Logout removes certificate and key for the currently logged in user from
+// the filesystem and agent.
 func (tc *TeleportClient) Logout() error {
-	return trace.Wrap(tc.localAgent.DeleteKey())
+	err := tc.localAgent.DeleteKey()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	return nil
+}
```

Comment:
wonder why you decided to expand the one-liner this way

Reply:
Whenever I am reading code I always miss the one-liners since they don't follow the typical Go pattern calling a function then checking the return value.